portable/CMakeLists.txt

```diff
@@ -1,177 +1,316 @@
 # FreeRTOS internal cmake file. Do not use it in user top-level project
 
 add_library(freertos_kernel_port STATIC
+    # 16-Bit DOS ports for BCC
     $<$<STREQUAL:${FREERTOS_PORT},BCC_16BIT_DOS_FLSH186>:
         BCC/16BitDOS/common/portcomn.c
         BCC/16BitDOS/Flsh186/port.c>
+
     $<$<STREQUAL:${FREERTOS_PORT},BCC_16BIT_DOS_PC>:
         BCC/16BitDOS/common/portcomn.c
         BCC/16BitDOS/PC/port.c>
+
+    # ARMv7-M port for Texas Instruments Code Composer Studio
     $<$<STREQUAL:${FREERTOS_PORT},CCS_ARM_CM3>:
         CCS/ARM_CM3/port.c
         CCS/ARM_CM3/portasm.asm>
+
+    # ARMv7-EM port for Texas Instruments Code Composer Studio
     $<$<STREQUAL:${FREERTOS_PORT},CCS_ARM_CM4F>:
         CCS/ARM_CM4F/port.c
         CCS/ARM_CM4F/portasm.asm>
+
+    # ARMv7-R port for Texas Instruments Code Composer Studio
     $<$<STREQUAL:${FREERTOS_PORT},CCS_ARM_CR4>:
         CCS/ARM_Cortex-R4/port.c
         CCS/ARM_Cortex-R4/portASM.asm>
+
+    # Texas Instruments MSP430 port for Texas Instruments Code Composer Studio
     $<$<STREQUAL:${FREERTOS_PORT},CCS_MSP430X>:
         CCS/MSP430X/port.c
         CCS/MSP430X/portext.asm>
+
+    # NXP (formerly Motorola, Freescale) Cold Fire and 68HCS12 ports for Code Warrior
     $<$<STREQUAL:${FREERTOS_PORT},CODEWARRIOR_COLDFIRE_V1>:
         CodeWarrior/ColdFire_V1/port.c
         CodeWarrior/ColdFire_V1/portasm.S>
+
     $<$<STREQUAL:${FREERTOS_PORT},CODEWARRIOR_COLDFIRE_V2>:
         CodeWarrior/ColdFire_V2/port.c
         CodeWarrior/ColdFire_V2/portasm.S>
+
     $<$<STREQUAL:${FREERTOS_PORT},CODEWARRIOR_HCS12>:
         CodeWarrior/HCS12/port.c>
+
+    # ARMv7-A port for GCC
     $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CA9>:
         GCC/ARM_CA9/port.c
         GCC/ARM_CA9/portASM.S>
+
+    # ARMv8-A ports for GCC
     $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CA53_64_BIT>:
         GCC/ARM_CA53_64_BIT/port.c
         GCC/ARM_CA53_64_BIT/portASM.S>
+
     $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CA53_64_BIT_SRE>:
         GCC/ARM_CA53_64_BIT_SRE/port.c
         GCC/ARM_CA53_64_BIT_SRE/portASM.S>
+
+    # ARMv6-M port for GCC
     $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CM0>:
         GCC/ARM_CM0/port.c>
+
+    # ARMv6-M / Cortex-M0 Raspberry PI RP2040 port for GCC
```

Comment:
Should we add this as people will mostly use this port with pico Cmake files which provides how to pull FreeRTOS - https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/portable/ThirdParty/GCC/RP2040/FreeRTOS_Kernel_import.cmake

Reply:
Either way is fine to me.

Reply:
To clarify, I just moved this from later on in the file so that it is near the other Cortex-M0 port. I don't think it's best to remove it since user projects may be depending on it.